dora/stats/admin.py

```diff
@@ -151,12 +152,36 @@ class OrientationEventAdmin(AnalyticsEventAdmin):
     list_display = [
         "date",
         "service",
+        "is_di",
```

Comment:
Je n'ai pas pu tester correctement la visualisation d'event de partage dans l'admin, mon front étant pété.
Mais tout à l'air ok côté code et fonctionne sur mon poste !

Reply:
Maintenant j'ai pu tester. Les infos sont ok dans l'admin, mais il y a apparemment des lenteurs. Je passe sur Silk

Reply:
à l’envoi? Oublie pas que les envois mailtrap prennent à peu près une seconde, ça pourrait être ça. Moi j’ai rien noté de plus a priori.